schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/Request.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.lsm.request;
+
+import org.apache.iotdb.lsm.context.RequestContext;
+
+import java.util.List;
+
+public abstract class Request<K, V, R> {
+
+  RequestType requestType;
```

Comment:
Since these bunch of abstract class ```Request``` and ```XXXRequest``` are stateless, make them as interface and it will be more flexible. The attribute ```requestType``` could be eliminated by setting default implementation of method ```getRequestType()```.